spec/shared_context/registration_shared.rb

```diff
@@ -0,0 +1,37 @@
+require 'elements/Elements'
+require 'rspec'
+require 'watir'
+require 'helper'
+
+shared_context 'Registration' do |username, surname, email, password|
```

Comment:
in case that you have multiple variables you can write this code in following way:

shared_context 'Registration' do |registration_data|

username = registration_data['username']
surname = registration_data['surname']
email = registration_data['email']
password = registration_data['password']